crates/server/src/config/mod.rs

```diff
@@ -226,12 +226,11 @@ impl ZoneConfig {
         Self {
             zone,
             zone_type,
-            file: Some(file),
             allow_update,
             allow_axfr,
             enable_dnssec,
             keys,
-            stores: None,
+            stores: Some(StoreConfig::File(FileConfig::new(file))),
```

Comment:
it doesn't have to be in this PR but IMO it would be clearer to rename this constructor to something like `new_with_zone_file` given that it always produces the `StoreConfig::File` variant.

Reply:
sounds good

Reply:
done in 34b5c49d